solitude/exceptions.py

```diff
@@ -13,6 +13,7 @@
 def custom_exception_handler(exc):
     # If you raise an error in solitude, it comes to here and
     # we rollback the transaction.
+    log.exception('Handling exception, about to roll back')
```

Comment:
I'm not sure that it's safe to do this because even though it logs the last exception in the system stack, Django might caught something innocuous (like a KeyError) already.